documentation/public/user-guide/introduction.md

```diff
@@ -0,0 +1,18 @@
+## What is Slang?
+
+Slang is intended to be a modular Solidity compiler, specifically targeting code analysis and developer tooling. This means servicing tools with domain-specific APIs and, in general, facilitating working with and analyzing the Solidity source code. If you're in the editor writing Solidity or performing linting or additional validation, there's a chance that you are, or could be, running Slang!
+
+## What Slang is not?
+
+First and foremost, it is not a replacement for `solc`, the standard Solidity compiler. We do not plan at the moment to support emitting EVM bytecode. Secondly, it is not meant to be used for formal verification of contracts or Solidity logic in general.
```

Comment:
discussion: distinction of production vs development codegen.

Reply:
Thanks; done in ae4b8969d6885a2acbcb16b221eadf6e7a885a69